src/joystick/windows/SDL_dinputjoystick.c

```diff
@@ -887,6 +904,10 @@ SDL_DINPUT_JoystickRumble(SDL_Joystick * joystick, Uint16 low_frequency_rumble,
     /* Scale and average the two rumble strengths */
     Sint16 magnitude = (Sint16)(((low_frequency_rumble / 2) + (high_frequency_rumble / 2)) / 2);
 
+    if (dinput == NULL) {
```

Comment:
This shouldn't be called if dinput is NULL. If that's not the case, then that's a bug in the higher level code and should be fixed.